kickstart/scripts/omk-deploy.sh

```diff
@@ -16,9 +17,21 @@ deploy_omk_ubuntu() {
 deploy_omk_server() {
 	# install OMK Server
   from_github "https://github.com/AmericanRedCross/OpenMapKitServer" "$dst/OpenMapKitServer"
+
+  # fetch git submodules
+  wget -q -O /root/sources/pyxform.tar.gz "https://github.com/spatialdev/pyxform/archive/e486b54d34d299d54049923e03ca5a6a1169af40.tar.gz"
+  tar -zxf /root/sources/pyxform.tar.gz -C "$dst/OpenMapKitServer/odk/pyxform" --strip=1
+
+  # install python packages for pyxform
+  cd $dst/OpenMapKitServer && pip install -r requirements.txt
```

Comment:
Could this be done in a virtualenv, similar to https://github.com/AmericanRedCross/posm-build/blob/master/kickstart/scripts/fieldpapers-deploy.sh#L144, to keep OMK Server's dependencies isolated?

(`PATH` and `VIRTUAL_ENV` would need to be set via upstart, per https://github.com/AmericanRedCross/posm-build/blob/master/kickstart/etc/fp-tasks.upstart#L8 and https://github.com/AmericanRedCross/posm-build/blob/master/kickstart/etc/fp-tasks.upstart#L17)